### ISSUE TITLE
[Feature] Show a preview to follow a contact

### ISSUE THREAD

**Issue by Clarin98** [redmatrix/hubzilla#987]
As Diaspora* project partially did, could be really "time-saving" to see a profile preview when you pass your mouse over the contact name, even letting you to follow that person in that "pop-cloud". Thanks for your work friends.

**Comment by zotlabs** [comment]
For supported networks you can follow the person from the 'gear' menu of an article they have authored. 

The profile popup is complicated by Hubzilla's extended network federation. The only information we require for federation with any message source is a name, a photo and a link. These are already provided in most places that the channel is referenced. (Mentions in posts do not contain the photo). This minimal requirement allows us to federate with any message source. For some message sources, all we have is a name - and the link and photo are generated from defaults. So we can even federate with email and random webapps. It isn't clear where we are suppose to scrape up the additional information that you might expect to find in such a profile popup, or if it is even possible in some cases. 

Diaspora on the other hand requires you to use the Diaspora protocol to interact on that network. There are no exceptions. This compliance involves a large number of dependencies and rules, including the presence of XML profiles with some standard fields.

**Comment by Clarin98** [comment]
Yeah, I want to complement my idea. I have seen that option that you mention, but is necessary to reload the web (going to the new contact's page) every time that you follow a new contact? In diaspora case, sometimes you can follow somebody passing the mouse over the name, clicking to add to an aspect, and after seeing a notice that you follow that person, but it doesn't charge again the page.

**Comment by zotlabs** [comment]
That would actually be pretty easy.
